src/mame/drivers/gaelco2.cpp

```diff
@@ -513,11 +513,11 @@ ROM_START( saltcrdi ) // REF 970410
 	ROM_LOAD( "bi-41-bank1-27c1001.u41",   0x01a0000, 0x0020000, CRC(c4d24254) SHA1(e6ff7624e628dc6ace11a50b6ff89812844b52c5) )
 	ROM_FILL(                              0x0220000, 0x0020000, 0x00 )         /* to decode GFX as 5bpp */
 
-	ROM_REGION( 0x1000, "iomcu", 0 ) // on IO board
-	ROM_LOAD( "st62t15c6",   0x0000, 0x1000, NO_DUMP ) // size?
+	ROM_REGION( 0x0800, "iomcu", 0 ) // on IO board
+	ROM_LOAD( "st62t15c6",   0x0800, 0x1000, NO_DUMP ) // 2KBytes internal ROM
```

Comment:
You're loading the ROM past the ROM_REGION length here. Pretty sure it will fail validation.

Reply:
I've fixed it with a second commit. Thanks.